apps/prairielearn/src/models/course.ts

```diff
@@ -12,3 +24,50 @@ export async function selectCourseById(course_id: string): Promise<Course> {
     CourseSchema,
   );
 }
+
+export async function selectAuthorizedCourses({
+  user_id,
+  is_administrator,
+  current_course,
```

Comment:
Nit: the course instance list function receives an ID. Here you only need an ID, I think for consistency this parameter could be an ID.

Reply:
In general we are moving towards passing around whole row objects rather than IDs to improve type safety (versus effectively untyped IDs).

Reply:
Changed in 5f70f9e.